### ISSUE TITLE
I can't see the ssh url when I click clone in TFS

### ISSUE THREAD

**Issue by alexisCata** [MicrosoftDocs/vsts-docs#1171]
How can I see the ssh url? When I click clone I only can see the HTTP url

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 64e85ff9-9add-d15c-e92f-02d7dd05e0f2
* Version Independent ID: 0cbd7427-6acd-3cfc-1a3b-8bf5ca60f3bd
* Content: [Connect to your Git repos with SSH](https://docs.microsoft.com/en-us/vsts/git/use-ssh-keys-to-authenticate?view=vsts)
* Content Source: [docs/git/use-ssh-keys-to-authenticate.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/git/use-ssh-keys-to-authenticate.md)
* Product: **devops**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by mmitrik** [comment]
@alexisCata Can you share a screenshot of what you see when you click Clone in the web view?

**Comment by alexisCata** [comment]
![untitled](https://user-images.githubusercontent.com/6735063/42038859-43a9df8c-7aec-11e8-9e4b-02f63f9c8794.jpg)

**Comment by alexisCata** [comment]
Microsoft Visual Studio Team Foundation Server
Version 15.112.26307.0

So it's TFS 2017 Update 1 RTM 

Should I activate it or something?

**Comment by mmitrik** [comment]
Yes, you'll need to configure it in the admin console.  Check out this doc:  https://docs.microsoft.com/en-us/tfs/server/install/single-server#configuration

Interesting bit:
> 
> If you select the New Deployment - Advanced scenario, you will additionally need to choose:
> 
> ...
> * Whether to enable SSH, along with the port it should listen on. The default value used in the Basic scenario is to configure SSH on port 22.

It might be that someone didn't enable SSH during setup.  You can also check on the server to see if `TeamFoundationSshService` is running, and then use the admin console to enable SSH and specify the desired port.

**Comment by Gladskih** [comment]
@alexisCata did you find a workaround for this bug?

**Comment by alexisCata** [comment]
@Gladskih finally I used http
